app/src/main/java/com/github/sdpcoachme/data/UserInfo.kt

```diff
@@ -11,8 +11,9 @@ data class UserInfo(
     val location: UserLocation,
     val coach: Boolean,
     val sports: List<Sports> = emptyList(),
-    val events: List<Event> = emptyList()
+    val events: List<Event> = emptyList(),
+    val chatContacts: List<String> = listOf(), // TODO: order based on message timestamp
```

Comment:
Check if this TODO comment can be removed.

Reply:
Thank you for spotting this. Indeed, this todo is a little unclear but this feature has been implemented (it refers to ordering the contacts so the the most recently messaged contacts come first). I will delete it.